examples/chern_insulators/haldane_C3.py

```diff
@@ -57,6 +57,10 @@ def init_sites(self, model_params):
         fs = FermionSite(conserve=conserve)
         gs = GroupedSite([fs, fs], labels=['A', 'B'], charges='same')
         gs.add_op('Ntot', gs.NA + gs.NB, False)
+
+        print(sorted(gs.opnames))
```

Comment:
We should remove any print statements, or at least enclose them in a `if verbose > 1:` clause

Reply:
These print statement have been removed.